extensions/wikia/AssetsManager/config.php

```diff
@@ -333,6 +333,7 @@
 	'skin' => [ 'oasis' ],
 	'type' => AssetsManager::TYPE_JS,
 	'assets' => [
+		'//extensions/wikia/AdEngine/js/video/player/porvata/floater.js',
```

Comment:
I believe floater is needed on Mercury as well or require.optional in porvata module